conf/messages.en

```diff
@@ -376,6 +377,7 @@ departure.declaration.errors.message.paragraph3.link = New Computerised Transit
 departure.declaration.errors.message.paragraph3.suffix = for help understanding the errors (opens in a new tab).
 departure.declaration.errors.message.hyperlink = Make another departure declaration
 departure.declaration.errors.message.hyperlink.text = Make another departure declaration
+departure.declaration.next.hidden.text = Next page of departure declarations
```

Comment:
Appreciate the ticket doesn't specify it but I'd have thought if we're doing this for the `Next` button we should do it for the `Previous` button as well?